app/src/main/res/layout/tv_shows_fragment.xml

```diff
@@ -1,13 +1,20 @@
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.core.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Тут лишний NestedScrollView, у нас же и так есть RecyclerView который будет скролится с сериалами. Лишний контейнер. 


> It's generally not a good idea to do this because you lose the main advantage of RecyclerView which is recycling. When RV is wrapped in NSV, it has to load everything it holds, even things that aren't visible.

https://www.reddit.com/r/androiddev/comments/1031p9c/comment/j2wbs44/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button